applications/minotari_console_wallet/src/automation/commands.rs

```diff
@@ -1007,7 +1007,7 @@ pub async fn command_runner(
 
                 let mut metadata_signature = Signature::default();
                 match key_manager_service
-                    .sign_with_nonce_and_message(
+                    .sign_with_challenge_and_message(
```

Comment:
```suggestion
                    .sign_with_nonce_and_challenge(
```
Previous name was incorrect, but this is more incorrect. 

in `sig = r +e*k`
you have 
r = nonce
e = challenge

where challenge = hash(R||K||message)